Server/JokesOnYou.Web.Api/JokesOnYou.Web.Api.csproj

```diff
@@ -5,26 +5,19 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <DocumentationFile>C:\Users\nicco\source\repos\JokesOnYou\Server\JokesOnYou.Web.Api\JokesOnYou.Web.Api.xml</DocumentationFile>
+    <DocumentationFile>JokesOnYou.Web.Api.xml</DocumentationFile>
     <NoWarn>1701;1702;1591;</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.4" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="5.0.4" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SQLite" Version="5.0.4" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="5.0.4">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.1.1" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Update="JokesOnYou.db">
-      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
```

Comment:
was this changed to copy if newer or just deleted?

Reply:
There's no need to copy it anywhere? even if it gets deleted the program will generate a new db and put users inn it

Reply:
its to avoid issues in the future, its very common. for example when publishing an app to a folder we need it to get the db.

copy paste from google:

`Copy always: This option works best when a developer intent is to have a clean slate to ensure their code functions correctly rather than having to clean up data on each build.

Copy if Newer: Use this option when the intent is to ensure changes to a database or a file persist between builds. Unlike [Copy always], [Copy if Newer] will leave files untouched unless there is a change to the file. For example, adding a new field or changing the field type in a database table will trigger a onetime copy, the same for working with any file, make a change and there will be a onetime copy.

This also means any data in a file within the build folder will be overwritten.

Do not Copy: As the option indicates, the file will not be copied. Note that changing from one of the other options will cause the file to be removed from the build folder. If the file in the build folder is needed then first, create a copy of the file before changing Copy to Output Directory to [Do not Copy].`